tests/syntax/fixtures_structure/placeable_at_eol.json

```diff
@@ -3,40 +3,75 @@
   "body": [
     {
       "type": "Message",
-      "span": {
-        "type": "Span",
-        "start": 0,
-        "end": 130
-      },
       "annotations": [],
       "id": {
         "type": "Identifier",
-        "name": "key"
+        "name": "key",
+        "span": {
+          "type": "Span",
+          "start": 0,
+          "end": 3
+        }
       },
       "value": {
         "type": "Pattern",
         "elements": [
           {
             "type": "TextElement",
-            "value": "A multiline message with a "
+            "value": "A multiline message with a ",
+            "span": {
+              "type": "Span",
+              "start": 5,
+              "end": 39
+            }
           },
           {
             "type": "MessageReference",
             "id": {
               "type": "Identifier",
-              "name": "placeable"
+              "name": "placeable",
+              "span": {
+                "type": "Span",
+                "start": 39,
+                "end": 48
+              }
+            },
+            "span": {
+              "type": "Span",
+              "start": 39,
+              "end": 48
```

Comment:
The message reference should cover { foo }, I think, and the identifier foo, so I expect those two to have different spans?